cloudbuild.yaml

```diff
@@ -1,11 +1,12 @@
 # See https://cloud.google.com/cloud-build/docs/build-config
 
 # this must be specified in seconds. If omitted, defaults to 600s (10 mins)
-timeout: 1200s
+timeout: 9000s
 # this prevents errors if you don't use both _GIT_TAG and _PULL_BASE_REF,
 # or any new substitutions added in the future.
 options:
   substitution_option: ALLOW_LOOSE
+  machineType: 'E2_HIGHCPU_8'
```

Comment:
good headroom to build the images. we use bigger machine types in many of our cloudbuild scripts